db/migrate/20161018134050_create_widgets.rb

```diff
@@ -0,0 +1,23 @@
+class CreateWidgets < ActiveRecord::Migration
+  def change
+    create_table :widgets do |t|
+      t.integer :widget_type, null: false
+      t.integer :position, null: false
+      t.text :properties
```

Comment:
Yes, but that's PostgreSQL specific. It is still interpreted as JSON, as specified in widget model, and by doing it this way, it works for all DB-s.